### PR TITLE
Newsletter-215: Translate to Hindi

### PR DESCRIPTION
**Pull request by shaavan**

### CODE REVIEW

File: _posts/hi/newsletters/2022-08-31-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  न हों। PSBT को फिर उस वॉलेट के साथ साझा किया जा सकता है जो उस पर हस्ताक्षर कर सकता है।
+
+- [Eclair #2275][] एक [दोहरी वित्त पोषित][topic dual funding] LN सेटअप लेनदेन शुल्क बंपिंग के लिए
+  समर्थन जोड़ता है। PR नोट करता है कि, इस PR के साथ, "दोहरी फंडिंग पूरी तरह से Eclair द्वारा समर्थित है!"
+  हालांकि यह भी नोट करता है कि डिफ़ॉल्ट रूप से दोहरी फंडिंग अक्षम है और [Core Lightning के साथ क्रॉस संगतता][news143 cln df]
+  के लिए परीक्षण भविष्य में जोड़े जाएंगे।
+
+- [Eclair #2387][] [signet][topic signet] के लिए समर्थन जोड़ता है।
+
+- [LDK #1652][][onion message] के लिए अद्यतन समर्थन [topic onion messsage] * उत्तर पथ *
```
Comment (bitschmidty) [inline _posts/hi/newsletters/2022-08-31-newsletter.md:106]: @shaavan the [onion message] ...... [topic onion messsage] needs to be corrected so it is clear what the link is and what is the link text. Im unsure of the language so need your help with that.

Also note "onion message" vs "onion mess**s**age"

**Review by bitschmidty — CHANGES_REQUESTED** [review]

**Comment by shaavan** [comment]
Updated from 8b0a023ef9a4ed34f207ba98f44ff523e19ca4d3 to 7af392b4d5b17e634725b2d0402144040674e4ad ([pr850.01][https://github.com/shaavan/bitcoinops.github.io/commits/pr850.01] -> [pr850.02][https://github.com/shaavan/bitcoinops.github.io/commits/pr850.02], [diff][https://github.com/shaavan/bitcoinops.github.io/compare/pr850.01..pr850.02])

Changes:
- Fixed two mistakes with `[link tag][link]` not correctly written. Thank you @bitschmidty for catching it!

File: _posts/hi/newsletters/2022-08-31-newsletter.md (hunk 1 above)
Comment (shaavan) [reply _posts/hi/newsletters/2022-08-31-newsletter.md]: Hi @bitschmidty, I was on a mini break, so I got your message today. Apologies for the late reply.

I figured the link tag was not correctly placed; hence the problem was not detected while running the `make production -j8`. There was also a similar mistake following it. So I have corrected both of them.
Thanks, @bitschmidty, for catching them!

**Review by bitschmidty — APPROVED** [review]
Looks good, thanks @shaavan !

**Pull request merged**